controls/roles/manage-service/molecule/ssv-lighthouse/prepare.yml

```diff
@@ -27,6 +38,24 @@
     - include_role:
         name: "configure-firewall"
 
+    - name: Create SSV config file
+      file:
+        path: "/opt/app/services/{{ ssv_service }}/data/blox/ssv"
+        state: directory
+        recurse: yes
+        owner: "2000"
+        group: "2000"
+      become: yes
+
+    - name: Write SSV configuration
```

Comment:
this looks like a douplicate to https://github.com/stereum-dev/ethereum-node/pull/223/files#diff-2bbb481048e42ab2451b5701b233c6ba21b097f70191bb6f7bc8d934a9c5a1c8R56 to me, what's the purpose of doing it twice?

Reply:
It's not duplicated. So it creates blox-ssv `config.yaml` file with its content, but without ssv-keys. So ssv-keys are written afterwards